src/student.js

```diff
@@ -0,0 +1,18 @@
+let id = 1
+
+function setId(newId) {
+  id = newId
+}
+
+class Student {
```

Comment:
What you've done here is introduce complexity by having something external to a class be able to mutate state related to the class. Someone could import this setId function anywhere and set it to a string, a number, a boolean, an object, and that would break everything that uses the Student class.

A "Manager" class should be the thing to _manage_ things, so I would store an ID counter in there. Then, since the Manager should be the class used to create and enrol students, the Manager class can manage the ID counter